src/systempolicy/systemPolicy.go

```diff
@@ -936,6 +938,17 @@ func PopulateSystemPoliciesFromSystemLogs(sysLogs []types.KnoxSystemLog) []types
 		// get k8s pods
 		pods := cluster.GetPods(clusterName)
 
+		// check for vm type and if exist add default to podname
+		for _, sysLog := range sysLogs {
+			if sysLog.Namespace == types.PolicyDiscoveryHost {
+				pods = append(pods, types.Pod{
+					Namespace: types.PolicyDiscoveryHost,
+					PodName:   "default",
+				})
+				break
```

Comment:
```suggestion
```
Applying a break here would result in only the first syslog converted!!

Reply:
Not actually @nyrahul .
This break exits only from the inner for loop. 

When this if matches, it exits only from `for _, sysLog := range sysLogs {`